src/gaxios.ts

```diff
@@ -55,7 +74,6 @@ function loadProxy() {
   }
   return proxy;
 }
-loadProxy();
```

Comment:
So thinking back through this - I think I was front-loading this call so that the `require` call would always happen on app start, instead of potentially happening during a request and synchronously blocking the thread.  By removing this, we could introduce a slower HTTP call here, and kind of degrade performance at a random point in app execution, no?

Reply:
(sorry @bcoe I know you asked earlier, but it didn't make sense until I re-looked at the codes)

Reply:
Just had a discussion with @sofisl about a potential refactor here, to get the best of both worlds.